Assets/GothicVR/Shaders/Unlit-AlphaToCoverage.shader

```diff
@@ -0,0 +1,86 @@
+Shader "Unlit/Unlit-AlphaToCoverage"
```

Comment:
Hm. Maybe this shader will work without setting it to "Always Included Shaders".
I thought we need to put it inside _Resources_ folder to be found at runtime. Will it work for Standalone builds?

Reply:
It is included in always included shaders. 😘 https://github.com/GothicVRProject/GothicVR/pull/226/commits/31ef83133933de5f6570e9e58f22b0b68d0ca0b4#diff-d6b5b7d6a42d4e04b725021f38ab5029472e26932c63c9ba1a03a3bee95a9a1d